src/platform/pdf/templates/veteran_status_new.js

```diff
@@ -0,0 +1,305 @@
+/**
+ * Proof of Veteran Status PDF template.
+ *
+ * NB: The order in which items are added to the document is important,
+ * and thus PDFKit requires performing operations synchronously.
+ */
+/* eslint-disable no-await-in-loop */
+
+import { MissingFieldsException } from '../utils/exceptions/MissingFieldsException';
+import {
+  createAccessibleDoc,
+  createHeading,
+  registerVaGovFonts,
+} from './utils';
+
+const config = {
+  margins: {
+    top: 30,
+    bottom: 40,
+    left: 40,
+    right: 40,
+  },
+  headings: {
+    H1: {
+      font: 'Bitter-Bold',
+      size: 20,
+    },
+    H2: {
+      font: 'SourceSansPro-Bold',
+      size: 13,
+    },
+  },
+  paragraph: {
+    font: 'SourceSansPro-Regular',
+    size: 16,
+  },
+  text: {
+    boldFont: 'SourceSansPro-Bold',
+    font: 'SourceSansPro-Regular',
+    size: 9,
+  },
+  disclaimer: {
+    font: 'SourceSansPro-Regular',
+    size: 9,
+  },
+  instruction: {
+    font: 'SourceSansPro-Regular',
+    size: 12,
+  },
+};
+
+// Reusable function to fetch and return an image as a base64 data URL
+const fetchImage = async url => {
+  const fetchedImage = await fetch(url);
+  const contentType = fetchedImage.headers.get('Content-type');
+  const base64Data = Buffer.from(await fetchedImage.arrayBuffer()).toString(
+    'base64',
+  );
+  return `data:${contentType};base64,${base64Data}`;
+};
+
+const validate = data => {
+  const requiredFields = ['fullName', 'latestService']; // If there is no latestService, there is also no DoD ID
```

Comment:
This 'new' PDF template requires a `latestService` string instead of a service history array.